src/Pact/Analyze/Feature.hs

```diff
@@ -80,6 +82,12 @@ data Feature
   | FCeilingRound
   | FFloorRound
   | FModulus
+  -- Bitwise operators
+  | FBitwiseAnd
```

Comment:
For a typical builtin, we need to define three almost equivalent things:
* The operation that goes in our executable AST (`BitwiseAnd`)
* The feature (part of this enum) (`FBitwiseAnd`)
* The string name of the feature (`SBitwiseAnd`)

`PAT` defines a unidirectional pattern synonym between the latter two. `bitwiseOpP` establishes a connection between the first and last.